pytorch_lightning/utilities/cloud_io.py

```diff
@@ -42,7 +47,7 @@ def get_filesystem(path: Union[str, Path]):
     return LocalFileSystem()
 
 
-def atomic_save(checkpoint, filepath: str):
+def atomic_save(checkpoint: Any, filepath: str) -> None:
```

Comment:
is `checkpoint` Any or Dict[str, Any] ? 

similarly: is `filepath` `str` or `Union[str, Path]` ?

Reply:
@ananthsub According to the original doc, `filepath` for `fsspec.open` should be either `str` or `list` but `fsspec.Path` seems to me to work fine as well.
I agree, `checkpoint` should be rather `Dict[str, Any]` as you suggested as well.